seeds/reviewSeeds.json

```diff
@@ -0,0 +1,92 @@
+[
+ {
+          "rating": 2,
+          "review":"Dwight was on awesome until I felt he got comfortable in our sessions. In one session we had, Dwight could tell that I had not studied the night prior. He told me if I did not have time to study then he did not have time to waste on me that could be better spent with M on this years beet harvest.",
+          "daytime": false,
+          "nights": true,
+          "weekends": false,
+          "user_id": 1,
+          "tutor_id": 1
+  },
+  {
+    "rating": ,
+    "review": "",
+    "daytime": false,
+    "nights": true,
+    "weekends": false,
+    "user_id": ,
```

Comment:
Same here as above (there are a few instances in this file that need some data).